packages/portal-proto/src/features/set-operations/CountButtonWrapperForSetsAndCases.tsx

```diff
@@ -137,6 +137,8 @@ const CountButtonWrapperForSetsAndCases: React.FC<
     return await createSet({
       // TODO: possibly add error handling
       filters: filters,
+      intent: "user",
+      set_type: "frozen",
```

Comment:
i think ephemeral here if this is from set operations. for gene/ssm sets, they (at least for now) are not expected to stay around forever (plus they show up in manage sets). for cohorts, if they end up being saved (which i expect them to be), the backend will change them to frozen.

Reply:
should we distinguish between gene/ssm sets and case sets here?

Reply:
This is just for case sets/cohorts. The gene and mutation sets use the <SaveSelectionModal /> component